resource/interface_factories.cpp.em

```diff
@@ -0,0 +1,297 @@
+// generated from ros1_bridge/resource/interface_factories.cpp.em
+
+@###############################################
+@#
+@# Factory template specializations for a
+@# specific message type
+@#
+@# EmPy template for generating
+@# <pkgname>__<type>__<interfacename>__factories.cpp
+@#
+@###############################################
+@# Start of Template
+@#
+@# Context:
+@#  - ros2_package_name (str)
+@#    The ROS 2 package name of this file
+@#  - mappings (list of ros1_bridge.Mapping)
+@#    Mapping between messages as well as their fields
+@###############################################
+@
+@{
+from ros1_bridge import camel_case_to_lower_case_underscore
+}@
+#include "rclcpp/rclcpp.hpp"
```

Comment:
Nit:

```
#include <algorithm>
#include <memory>
#include <string>
```

Reply:
Added `algorithm` in a1f6a18acedcec9f23b405a12f001f6eca21ac0a. For the other two same as https://github.com/ros2/ros1_bridge/pull/183#discussion_r275847874.